src/I18n/TranslatorRegistry.php

```diff
@@ -76,7 +76,7 @@ public function setCacher(CacheEngine $cacher)
      * @param string $name The translator package to retrieve.
      * @param string|null $locale The locale to use; if empty, uses the default
      * locale.
-     * @return \Aura\Intl\TranslatorInterface A translator object.
+     * @return \Aura\Intl\TranslatorInterface|null A translator object.
```

Comment:
There is no such thing as `...|void`. It cannot exist by definition.
As to the requirement of the return null statement, that is debatable, even though I agree with you that it would be cleaner, and less "yellow" marked in IDEs :)

But in this case here these discussions are irrelevant to the changes.
Here we do have a clear return statement, and a dangerous one on top, which we need to declare and communicate as such.